inc/formatconvert.class.php

```diff
@@ -2050,7 +2050,9 @@ static function networkequipmentInventoryTransformation($array) {
                                                     'SYSDESCR' => 'sysdescr',
                                                     'MODEL'    => 'model',
                                                     'IP'       => 'ip',
-                                                    'SYSMAC'   => 'mac',
+                                                    //'SYSMAC'   => 'mac',
+                                                    'SYSMAC'   => 'sysmac',
+                                                    'MAC'      => 'mac',
```

Comment:
Not agree with that, if we are in `CDP`, we don't have the `MAC` field in the XML, see http://fusioninventory.org/documentation/dev/spec/protocol/netinventory.html